models/classes/runner/session/TestSession.php

```diff
@@ -193,18 +193,12 @@ public function getItemAttemptTag(RouteItem $routeItem)
      * Initializes the timer for the current item in the TestSession
      *
      * @param $timestamp
+     * @deprecated Need to use startItemTimer() method
      * @throws \oat\taoTests\models\runner\time\InvalidDataException
      */
     public function initItemTimer($timestamp = null)
     {
-        if (is_null($timestamp)) {
-            $timestamp = microtime(true);
-        }
-
-        // try to close existing time range if any, in order to be sure the test will start or restart a new range.
-        // if the range is already closed, a message will be added to the log
-        $tags = $this->getItemTags($this->getCurrentRouteItem());
-        $this->getTimer()->start($tags, $timestamp)->save();
```

Comment:
The purpose of `initItemTimer` is not to start a timer, but to close a previously opened timer interval in case there is one. As you have already identified, it was modified [here](https://github.com/oat-sa/extension-tao-testqti/pull/1632/files#diff-d55499ceda4a30cf6d0b11c132f9440cc8aa9603c56a387bfed87cf22e6aaf2eR204) and that problem should be addressed as well. I agree the name of the method is not clear.